bitcoin/src/psbt/error.rs

```diff
@@ -57,7 +57,7 @@ pub enum Error {
     /// Unable to parse as a standard sighash type.
     NonStandardSighashType(u32),
     /// Parsing errors from bitcoin_hashes
-    HashParse(hashes::Error),
+    HashParse(hashes::FromSliceError),
```

Comment:
Calling this one `HashParse` looks kinda strange now but this is for a different PR.

Reply:
Would be nice to come up with a naming convention for error variants that wrap single usecase errors, I can't think of an exact example right now but over the last few weeks I've been unhappy with my naming. Will mention again later.

Reply:
I'd just call it `Hash` or `InvalidHash`, since this is not parsing.

Reply:
#1888